k8s.io/configmap-nginx.yaml

```
@@ -235,9 +235,9 @@ data:
           rewrite ^/bot-commands$    https://prow.k8s.io/command-help redirect;
           rewrite ^/calendar$        https://www.k8s.dev/calendar redirect;
           rewrite ^/github-labels$   https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md redirect;
-          rewrite ^/good-first-issue$ https://github.com/search?q=org%3Akubernetes+org%3Akubernetes-sigs+org%3Akubernetes-csi+org%3Akubernetes-client+is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+no%3Aassignee&type=Issues redirect;
-          rewrite ^/help-wanted$     https://github.com/search?q=org%3Akubernetes+org%3Akubernetes-sigs+org%3Akubernetes-csi+org%3Akubernetes-client+is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+no%3Aassignee&type=Issues redirect;
-          rewrite ^/needs-ok-to-test$ https://github.com/search?q=org%3Akubernetes+org%3Akubernetes-sigs+org%3Akubernetes-csi+org%3Akubernetes-client+is%3Aopen+is%3Apr+label%3Aneeds-ok-to-test+label%3A%22cncf-cla%3A+yes%22+-label%3Aneeds-rebase&type=Issues redirect;
+          rewrite ^/good-first-issue$ https://github.com/issues?q=org%3Akubernetes+org%3Akubernetes-sigs+org%3Akubernetes-csi+org%3Akubernetes-client+is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+no%3Aassignee&type=Issues redirect;
```

nit: we can remove the `is:issue` search keyword since we are using `/issues` here.

```suggestion
          rewrite ^/good-first-issue$ https://github.com/issues?q=org%3Akubernetes+org%3Akubernetes-sigs+org%3Akubernetes-csi+org%3Akubernetes-client+is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+no%3Aassignee redirect;
```

---

Fun fact: you can get both issues and PRs from the /issues and /pulls endpoint.  I'm going to assume this is the legacy of pull requests being issues.

e.g.:
- https://github.com/issues?utf8=%E2%9C%93&q=involves%3Aspiffxp+archived%3Afalse+sort%3Aupdated-desc+
- https://github.com/pulls?utf8=%E2%9C%93&q=involves%3Aspiffxp+archived%3Afalse+sort%3Aupdated-desc+

So while the `type=` is good to drop, we probably want to keep the `is:issue` and `is:pr` stuff

---

gah 🤦